dev/MRTCore/packaging/MrtCore.PriGen.targets

```diff
@@ -1,18 +1,13 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="15.0">
 
-  <!-- With .NET 5 and above, the propery TargetPlatformIdentifier is not UAP but rather Windows.
-       Likewise, for other console apps, TargetPlatformIdentifier is Windows.
-       PRI generation in VS apps was developed for UWPs originally. Some of the binaries involved assume
+  <!-- PRI generation in VS apps was developed for UWPs originally. Some of the binaries involved assume
        that the app is a UWP. Change the value provided to said binaries to UAP. -->
   <PropertyGroup>
     <TargetPlatformIdentifierAdjusted>$(TargetPlatformIdentifier)</TargetPlatformIdentifierAdjusted>
 
-    <!-- Any .NET 5 or greater version is valid here! -->
-    <TargetPlatformIdentifierAdjusted Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' and $([MSBuild]::VersionGreaterThanOrEquals($(TargetFrameworkVersion), '5.0'))">UAP</TargetPlatformIdentifierAdjusted>
```

Comment:
@huichen123 please see the updated PR description for the reason for this removal. Please let me know if you're OK with it.
Also @axelandrejs if you have time

Reply:
Been discussed in another comment thread.